tests/network/fortinet/fortigate/snmp/network-fortinet-fortigate-snmp.robot

```diff
@@ -9,412 +9,67 @@ Test Timeout        120s
+            ...       13   ${EMPTY}         ${EMPTY}          ${EMPTY}       ${EMPTY}                   ${EMPTY}                     ${EMPTY}                     ${EMPTY}                      ${EMPTY}          ${EMPTY}           ${EMPTY}         ${EMPTY}          0.5                  ${EMPTY}              WARNING: Link monitor 'MonitorWAN2' [vdom: root] [id: 2] packet loss: 1.000% - Link monitor 'MonitorWAN3' [vdom: root] [id: 3] packet loss: 100.000% | 'MonitorWAN1~root#linkmonitor.latency.milliseconds'=39.739;;;0; 'MonitorWAN1~root#linkmonitor.jitter.milliseconds'=0.096;;;0; 'MonitorWAN1~root#linkmonitor.packet.loss.percentage'=0;0:0.5;;0; 'MonitorWAN2~root#linkmonitor.latency.milliseconds'=46.446;;;0; 'MonitorWAN2~root#linkmonitor.jitter.milliseconds'=1.868;;;0; 'MonitorWAN2~root#linkmonitor.packet.loss.percentage'=1;0:0.5;;0; 'MonitorWAN3~root#linkmonitor.latency.milliseconds'=0.000;;;0; 'MonitorWAN3~root#linkmonitor.jitter.milliseconds'=0.000;;;0; 'MonitorWAN3~root#linkmonitor.packet.loss.percentage'=100;0:0.5;;0;
+            ...       14   ${EMPTY}         ${EMPTY}          ${EMPTY}       ${EMPTY}                   ${EMPTY}                     ${EMPTY}                     ${EMPTY}                      ${EMPTY}          ${EMPTY}           ${EMPTY}         ${EMPTY}          ${EMPTY}             0.5                   CRITICAL: Link monitor 'MonitorWAN2' [vdom: root] [id: 2] packet loss: 1.000% - Link monitor 'MonitorWAN3' [vdom: root] [id: 3] packet loss: 100.000% | 'MonitorWAN1~root#linkmonitor.latency.milliseconds'=39.739;;;0; 'MonitorWAN1~root#linkmonitor.jitter.milliseconds'=0.096;;;0; 'MonitorWAN1~root#linkmonitor.packet.loss.percentage'=0;;0:0.5;0; 'MonitorWAN2~root#linkmonitor.latency.milliseconds'=46.446;;;0; 'MonitorWAN2~root#linkmonitor.jitter.milliseconds'=1.868;;;0; 'MonitorWAN2~root#linkmonitor.packet.loss.percentage'=1;;0:0.5;0; 'MonitorWAN3~root#linkmonitor.latency.milliseconds'=0.000;;;0; 'MonitorWAN3~root#linkmonitor.jitter.milliseconds'=0.000;;;0; 'MonitorWAN3~root#linkmonitor.packet.loss.percentage'=100;;0:0.5;0;
+    
+Network Fortinet Fortigate SNMP list link monitor ${tc}
```

Comment:
It is preferable to split into one file per mode: one for link-monitor and one for list-link-monitor, eache file named after the mode that is tested.
--> link-monitor.robot & list-link-monitor.robot